thesis-data/experiments-data/experiment 1 -- varying approval rate/config 2/config.txt
{
    "global": {
        "num max complaints":               10,
        "max hearing rounds":               3,
        "aquaculture blocking radius":      25,
        "aquaculture damage radius":        100,
        "aquaculture damage proportion":    1,
        "aquaculture in blocked":           false
    },
    "world": {
        "structure": {
            "class": {
                "type":  "class",
                "class": "FisherSimulation.world.GridStructure"
            },
            "width":                       15,
            "height":                      15,
            "cell width":                  25,
            "cell height":                 25,
            "neighbourhood type":          "von_neumann"
        },
        "good spot frequency": 0.1
    },
    "interface": {
        "print messages": false
    },
    "fisherman": {
        "num":        20,
        "priorities": {
            "OwnProfits":                    10.0,
            "CommunityWealth":               2.0,
            "WildFishPrice":                 2.0,
            "FishingIndustryExisting":       3.0,
            "NaturalFishHealth":             4.0,
            "AquacultureIndustryExisting":   1.0
        },
        "learning mechanism": {
            "class": {
                "type":  "class",
                "class": "FisherSimulation.ga.Evolution"
            },
            "phenotype class": {
                "type":  "class",
                "class": "FisherSimulation.ga.FishermanVotingNN"
            },
            "genotype class": {
                "type":  "class",
                "class": "FisherSimulation.ga.FishermanNNGenotype"
            },
            "elitism":              3,
            "selection mechanism":  "rank selection",
            "crossover rate":       0.005,
            "mutation rate":        0.005,
            "genome mutation rate": 0.00005
            
        },
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.ga.FishermanVotingNN"
        }
    },
    "aquaculture": {
        "priorities":      {},
        "work efficiency": 10,
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "civilian": {
        "num":        0,
        "priorities": {},
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "tourist": {
        "num":        0,
        "priorities": {},
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "government": {
        "priorities":   {},
        "num licenses": 5,
        "decision mechanism class": {
            "type": "class",
            "class": "FisherSimulation.dm.ApproveProbability"
        },
        "complaint approval probability": 0.2
    },
    "municipality": {
        "priorities": {},
        "planning mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.dm.EverythingAquaculture"
        }
    }
}

thesis-data/experiments-data/experiment 1 -- varying approval rate/config 3/config.txt
{
    "global": {
        "num max complaints":               10,
        "max hearing rounds":               3,
        "aquaculture blocking radius":      25,
        "aquaculture damage radius":        100,
        "aquaculture damage proportion":    1,
        "aquaculture in blocked":           false
    },
    "world": {
        "structure": {
            "class": {
                "type":  "class",
                "class": "FisherSimulation.world.GridStructure"
            },
            "width":                       15,
            "height":                      15,
            "cell width":                  25,
            "cell height":                 25,
            "neighbourhood type":          "von_neumann"
        },
        "good spot frequency": 0.1
    },
    "interface": {
        "print messages": false
    },
    "fisherman": {
        "num":        20,
        "priorities": {
            "OwnProfits":                    10.0,
            "CommunityWealth":               2.0,
            "WildFishPrice":                 2.0,
            "FishingIndustryExisting":       3.0,
            "NaturalFishHealth":             4.0,
            "AquacultureIndustryExisting":   1.0
        },
        "learning mechanism": {
            "class": {
                "type":  "class",
                "class": "FisherSimulation.ga.Evolution"
            },
            "phenotype class": {
                "type":  "class",
                "class": "FisherSimulation.ga.FishermanVotingNN"
            },
            "genotype class": {
                "type":  "class",
                "class": "FisherSimulation.ga.FishermanNNGenotype"
            },
            "elitism":              3,
            "selection mechanism":  "rank selection",
            "crossover rate":       0.005,
            "mutation rate":        0.005,
            "genome mutation rate": 0.00005
            
        },
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.ga.FishermanVotingNN"
        }
    },
    "aquaculture": {
        "priorities":      {},
        "work efficiency": 10,
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "civilian": {
        "num":        0,
        "priorities": {},
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "tourist": {
        "num":        0,
        "priorities": {},
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "government": {
        "priorities":   {},
        "num licenses": 5,
        "decision mechanism class": {
            "type": "class",
            "class": "FisherSimulation.dm.ApproveProbability"
        },
        "complaint approval probability": 0.8
    },
    "municipality": {
        "priorities": {},
        "planning mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.dm.EverythingAquaculture"
        }
    }
}

config/config.js
{
    "global": {
        "num max complaints":               10,
        "max hearing rounds":               3,
        "aquaculture blocking radius":      25,
        "aquaculture damage radius":        50,
        "aquaculture damage proportion":    1,
        "aquaculture in blocked":           false
    },
    "world": {
        "structure": {
            "class": {
                "type":  "class",
                "class": "FisherSimulation.world.GridStructure"
            },
            "width":                       15,
            "height":                      15,
            "cell width":                  25,
            "cell height":                 25
        },
        "good spot frequency": 0.1
    },
    "interface": {
        "print messages": false
    },
    "fisherman": {
        "num":        20,
        "priorities": {
            "OwnProfits":                    10.0,
            "CommunityWealth":               2.0,
            "WildFishPrice":                 2.0,
            "FishingIndustryExisting":       3.0,
            "NaturalFishHealth":             4.0,
            "AquacultureIndustryExisting":   1.0
        },
        "learning mechanism": {
            "class": {
                "type":  "class",
                "class": "FisherSimulation.ga.Evolution"
            },
            "phenotype class": {
                "type":  "class",
                "class": "FisherSimulation.ga.FishermanVotingNN"
            },
            "genotype class": {
                "type":  "class",
                "class": "FisherSimulation.ga.FishermanNNGenotype"
            },
            "elitism":              3,
            "selection mechanism":  "rank selection",
            "crossover rate":       0.005,
            "mutation rate":        0.005,
            "genome mutation rate": 0.00005
            
        },
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.ga.FishermanVotingNN"
        }
    },
    "aquaculture": {
        "priorities":      {},
        "work efficiency": 10,
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "civilian": {
        "num":        0,
        "priorities": {},
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "tourist": {
        "num":        0,
        "priorities": {},
        "voting mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.vote.AlwaysApprove"
        }
    },
    "government": {
        "priorities":   {},
        "num licenses": 5,
        "decision mechanism class": {
            "type": "class",
            "class": "FisherSimulation.dm.ApproveProbability"
        },
        "complaint approval probability": 0.5
    },
    "municipality": {
        "priorities": {},
        "planning mechanism class": {
            "type":  "class",
            "class": "FisherSimulation.dm.EverythingAquaculture"
        }
    }
}